lib/streamlit/type_util.py

```diff
@@ -357,7 +357,7 @@ def is_sympy_expession(obj: object) -> TypeGuard[sympy.Expr]:
 
 def is_altair_chart(obj: object) -> bool:
     """True if input looks like an Altair chart."""
-    return is_type(obj, _ALTAIR_RE)
+    return is_type(obj, _ALTAIR_RE) or is_type(obj, "altair.vegalite.api.Chart")
```

Comment:
Just in case you want to exchange it here as well for `altair.Chart` but maybe it's on purpose.

```suggestion
    return is_type(obj, _ALTAIR_RE) or is_type(obj, "altair.Chart")
```

Reply:
Actually, the added part here isn't really required, so I removed it again.